DataFormats/Headers/include/Headers/DataHeader.h

```diff
@@ -264,6 +264,14 @@ struct Descriptor {
 
   bool operator==(const Descriptor& other) const {return itg == other.itg;}
   bool operator!=(const Descriptor& other) const {return not this->operator==(other);}
+
+  // explicitly forbid comparison with e.g. const char* strings
+  // use: value == Descriptor<N>("DESC") for the appropriate
+  // template instantiation instead
+  template<typename T>
```

Comment:
I will merge this, but shouldn't we delete all the comparisons with something which is not a `const Description&`?